packages/core/src/avatar/Avatar.tsx

```diff
@@ -54,12 +54,14 @@ export const Avatar = forwardRef<HTMLDivElement, AvatarProps>(function Avatar(
     src,
     size = DEFAULT_AVATAR_SIZE,
     style: styleProp,
-    fallbackIcon = <UserSolidIcon aria-label="User Avatar" />,
+    fallbackIcon,
     ...rest
   },
   ref,
 ) {
   const targetWindow = useWindow();
+  const { SemanticUserSolidIcon = UserSolidIcon } = useIcon();
+  const feedBackIcon = <SemanticUserSolidIcon aria-label="User Avatar" />;
```

Comment:
feedback naming becomes fallback here ?

then fallbackIcon prop is no longer used by most likely still needed ?

Reply:
it was an existing code, used as a fallback on line 107
`{children || avatarInitials || feedBackIcon}`

Reply:
You found a bug... see where the prop fallbackIcon is used in this code?

```
  /**
   * Icon to be used as a default item. Defaults to `UserIcon`
   */
  fallbackIcon?: ReactNode;
```

it is passed as props but not used ?
Then we have a feedbackIcon which looks to have been given the wrong name.

Do we want to deprecate the fallbackIcon prop, you could achieve the same with children ?

Reply:
sorry my Bad, it was fallbackIcon only, by mistake named it feedbackIcon. corrected now